src/test/java/com/gradle/develocity/bamboo/ArtifactoryMaven3TaskMavenOptsSetterTest.java

```diff
@@ -29,9 +32,9 @@ void orderIsSet() {
     @Test
     void mavenOpsAreSet() {
         List<SystemProperty> systemProperties = new ArrayList<>();
-        systemProperties.add(new SystemProperty("maven.ext.class.path", "test/path/gradle-enterprise-maven-extension-1.15.4.jar"));
-        systemProperties.add(new SystemProperty("gradle.scan.uploadInBackground", "false"));
-        systemProperties.add(new SystemProperty("gradle.enterprise.url", "url"));
+        systemProperties.add(MAVEN_EXT_CLASS_PATH_SYSTEM_PROPERTY.forValue("test/path/gradle-enterprise-maven-extension-1.15.4.jar"));
```

Comment:
Right ? 
```suggestion
        systemProperties.add(MAVEN_EXT_CLASS_PATH_SYSTEM_PROPERTY.forValue("test/path/develocity-maven-extension-1.15.4.jar"));
```

Reply:
Doesn't matter in a unit test but I've updated to using the Develocity Maven extension instead.

CHANGED (cc7b9c99)